services/manager/internal/service/watchers.go

```diff
@@ -134,7 +145,12 @@ func (g *GRPC) WatchForPendingDocuments(ctx context.Context) error {
 			continue
 		}
 
-		logger.Info().Msgf("Config has been successfully processed and moved to the %q state", manifest.Scheduled.String())
+		switch result {
+		case NotReady:
```

Comment:
Do we need to catch the `NotReady` case when it does nothing?

Reply:
No we don't, but the linter will catch it.